airflow/www/templates/airflow/dag_code.html

```diff
@@ -31,7 +31,7 @@ <h4>{{ title }}</h4>
 {% endblock %}
 
 {% block tail %}
-    {{ super() }}
+    {{ super() if dag else '' }}
```

Comment:
I don't see why this change is needed.

Reply:
Otherwise dag page rendering fails on accessing dag attributes when dag is None.